README.adoc

```diff
@@ -19,7 +19,11 @@ git clone https://github.com/redhat-cop/openshift-playbooks.git
 +
 ----
 cd openshift-playbooks
-docker-compose up -d
+docker run \
```

Comment:
Can you add a quickstart step around the docker build the builder image?

Reply:
@beelandc once merged, we will build the image and publish it in docker hub, so user's won't need to run a docker build.

Reply:
ah, gotcha